### ISSUE TITLE
Minecraft Repair command

### ISSUE THREAD

**Issue by PhantumOne** [mcMMO-Dev/mcMMO#3221]
I am running a single server with multiple worlds and using Pex for per-world commands but for some reason the mcMMO is turning off Minecraft repair and replacing it with the mcMMO repair. I noticed that the command for God mode in mcMMO is /mcgod, why did you not use /mcrepair for the mcMMO pugin and is there a way for me to modify the repair command in mcMMO thru the configs so I can get back the normal repair for my other worlds?

**Comment by PhantumOne** [comment]
I have negated the nodes: - -mcmmo.skills.repair, - -mcmmo.commands.repair, - -mcmmo.ability.repair.*,  - -mcmmo.ability.repair.all, - -mcmmo.commands.repair for all other worlds in Pex and negated  - -essentials.repair in my RPG world and I receive an error "This command cannot be performed in this world". I have examined each line of my permissions file and did not find any conflicting nodes in other Goups or World sections. I then negated all mcMMO commands and essentials commands in the default group then gave permissions to them in the Users world nodes and I still get the same message. If I remove the mcMMO plugin and nodes then the normal minecraft repair command works fine in all the worlds defined except in my RPG world where I have blocked it using - -essentials.repair node. So the conflict is in the plugin that for some reason is making the mcMMO command "repair" global preventing me from negating it even when I use the negate nodes.
